data/battle_anim_scripts.s

```diff
@@ -17876,6 +17876,57 @@ Move_PSYBLADE::
 Move_BLOOD_MOON::
 Move_MATCHA_GOTCHA::
 Move_IVY_CUDGEL::
```

Comment:
Can you move it up so it before Tera Blast? Otherwise all the animations above it will have the ivy cudgel animation

Reply:
Done